.docs/tutorials.rst

```diff
@@ -42,6 +42,18 @@ Contents:
    _notebooks/tutorial01_mf6_output
 
 
+MODFLOW 6 FloPy Plugin Tutorials
+--------------------------------
+
+Contents:
+
+.. toctree::
+   :maxdepth: 2
+
+   _notebooks/tutorial01_mf6_plugin
+   _notebooks/tutorial02_mf6_plugin
```

Comment:
It'd be nice to have a tutorial on how to override functionality in an existing package. The current set of tutorials shows how to use an existing plugin and how to develop one in the API package. Modifying existing package behavior is an important topic.